.github/scripts/pack-debian-apt.js

```diff
@@ -81,7 +81,8 @@ PATH=$PATH:$PWD/bin eval $(PATH=$PATH:$PWD/bin node -p "require('./package').scr
       await qq.chmod([workspace, 'usr/lib', config.dirname, 'bin', config.bin], 0o755);
       await qq.chmod([workspace, 'DEBIAN/postinst'], 0o755);
       await qq.x(`ln -s "../lib/${config.dirname}/bin/${config.bin}" "${workspace}/usr/bin/${pjson.oclif.bin}"`);
-      await qq.x(`dpkg --build "${workspace}" "${qq.join(dist, debArch(arch), `${versionedDebBase}.deb`)}"`);
+      // Use xz compression until Debian supports zstd (see issue #482)
+      await qq.x(`dpkg -Zxz --build "${workspace}" "${qq.join(dist, debArch(arch), `${versionedDebBase}.deb`)}"`);
```

Comment:
Getting error:
dpkg: error: unknown option -Z

Reply:
Thanks for testing this @sbansla. I just pushed a commit to use `dpkg-deb` instead of `dpkg` to do the build. That should resolve the unknown option error, but I'm not sure if the build will still work. Would you mind trying it again?